src/java.base/windows/native/libnet/NetworkInterface.c

```diff
@@ -672,11 +672,18 @@ jobject createNetworkInterface
         (*env)->SetObjectArrayElement(env, addrArr, addr_index, iaObj);
         addrs = addrs->next;
         addr_index++;
+        (*env)->DeleteLocalRef(env, iaObj);
+        (*env)->DeleteLocalRef(env, ibObj);
+        (*env)->DeleteLocalRef(env, ia2Obj);
```

Comment:
ia2Obj is not guaranteed to be set to a local reference, no? (if the address family is AF_INET6)

Reply:
You're right. I've corrected that and made the same last-use/deleteRef locality change as in the UNIX code. Thanks!